iree/compiler/Dialect/LinalgExt/IR/LinalgExtInterfaces.cpp

```diff
@@ -0,0 +1,28 @@
+// Copyright 2021 The IREE Authors
+//
+// Licensed under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include "iree/compiler/Dialect/LinalgExt/IR/LinalgExtInterfaces.h"
+
+namespace mlir {
+namespace iree_compiler {
+namespace linalg_ext {
+
+OpOperandVector::operator SmallVector<Value>() {
```

Comment:
Hmmm, not a fan of such implicit conversions. Very easy to not see that there is a copy happening here.

Reply:
This is copied from Linalg dialect. Most of interface methods rely on it.